### PR TITLE
[test/plugin] immigrate test httpasyncclient scenario.

### PR DESCRIPTION
**Pull request by aderm**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.
run test httpasyncclient scenario stat:
Scenarios: httpasyncclient-scenario, Testcases: 7, parallel_run_size: 1, Elapsed: 00:04:50

### CODE REVIEW

**Comment by wu-sheng** [comment]
Related to #3583

**Comment by aderm** [comment]
ok

**Comment by wu-sheng** [comment]
And next time, you put the execution time list in the PR description too. Please move this task. FYI @arugal

**Comment by wu-sheng** [comment]
Both of you. I think I find the workload2 is faster?

**Comment by aderm** [comment]
i have check, workload-1 is faster workload-2   Dozens of seconds

**Comment by aderm** [comment]
ok

File: test/plugin/scenarios/httpasyncclient-scenario/config/expectedData.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+registryItems:
+    applications:
+    - {httpasyncclient-scenario: nq 0}
+    instances:
+    - {httpasyncclient-scenario: 1}
+    operationNames:
+    - httpasyncclient-scenario: [/httpasyncclient/case/httpasyncclient, /httpasyncclient/back]
+segmentItems:
+- applicationCode: httpasyncclient-scenario
+  segmentSize: 4
```
Comment (dmsolr) [inline test/plugin/scenarios/httpasyncclient-scenario/config/expectedData.yaml]: Suggestion, `ge 4`
Sometimes, this will be called several times.

Comment (wu-sheng) [reply test/plugin/scenarios/httpasyncclient-scenario/config/expectedData.yaml]: Yes, agree. `ge 4` is more safe. @aderm Fix this?

Comment (aderm) [reply test/plugin/scenarios/httpasyncclient-scenario/config/expectedData.yaml]: ok

**Pull request merged**